recipes/imp/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set version = "2.15.0" %}
+
+package:
+  name: imp
+  version: {{ version }}
+
+source:
+  fn: imp-{{ version }}.tar.gz
+  url: https://integrativemodeling.org/{{ version }}/download/imp-{{ version }}.tar.gz
+  sha256: be2c13c95e4347f2fa4fe4f845a109464e9b95c6a417e286ba57aa79a7cd1a16
+  patches:
+    - clang-6-cpp14.patch
+    - cmake-w32-deps.patch
+    - sfmap-swap.patch
+    - anon-export-fix.patch
+    - bc-clustering.patch  # [osx]
+    - imp-directories.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: True  # [not win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - pkg-config   # [not win]
+    - swig
+    - cmake
+    - ninja
+  host:
+    - python
+    - numpy
+    - rmf
+    - ihm
+    - boost-cpp
+    - hdf5
+    # We need both the libprotobuf library and the Python wrappers
+    - protobuf
+    # We only need the library; we don't use the OpenCV Python wrappers
+    - libopencv
+    - eigen
+    - fftw
+    - gsl
+    - cgal-cpp
+    - mpich  # [not win]
+    - msmpi  # [win]
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    # RMF does not pin its exports, but we link against the C++ DSO
+    - {{ pin_compatible('rmf', max_pin='x.x') }}
+    - ihm
+    # CGAL code pulls in mpfr and gmp (on Linux/Mac) or mpir (on Windows)
+    - gmp   # [not win]
+    - mpir  # [win]
+    - mpfr
+    - libopencv
+    - boost-cpp
+    - fftw
+    - hdf5
+    - gsl
+    - libopenblas  # [linux]
+    - libprotobuf
+    - protobuf
```

Comment:
All of these except `boost-cpp` should be `host` dependencies.

Reply:
Thanks for reviewing! But I'm not sure what you mean - most of these dependencies (libopencv/fftw/hdf5/gsl/protobuf) *are* in the `host` section already. gmp/mpir/mpfr are not because they are pulled in by CGAL at build time, but not at runtime because CGAL is header-only; libopenblas/libprotobuf are pulled in by gsl/protobuf but are explicitly listed at runtime to satisfy the overlinking check. But I can certainly duplicate them in the `host` section if that is more typical conda-forge style?

Reply:
@isuruf Assuming you mean "yes, `host` dependencies should be a superset of `run` dependencies" I'll go ahead and make that change, then request a second review should the CI pass.

Reply:
`gmp` and other packages have `run_exports`. When you specify them in `host`, they are automatically added to `run` with the correct version range. So, they need to be only in `host`.